content/en/docs/collector/building/_index.md

```diff
@@ -9,3 +9,51 @@ also by custom components, that you develop and build on your own. Here you will
 find instructions on how to build some of those components. For additional
 details take a look into the documents contained within the
 [opentelemetry-collector-contrib repository](https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/README.md).
+
+There are several
+[classes](https://github.com/open-telemetry/opentelemetry-collector/blob/main/cmd/mdatagen/main.go#L184)
+of OpenTelemetry components, such as
+
+- `receivers` Scrapers and Listeners to ingest data
+- `exporters` Ways to export data to vendors or other tooling
+- `processors` Ways to process data in a pipeline
+- `connectors` Ways to [connect pipelines](./connector/) and form a
+  [DAG](https://en.wikipedia.org/wiki/Directed_acyclic_graph) for your data
+  pipeline
```

Comment:
Hmmm, I don't think people will really understand why having a DAG is helpful? I'd propose maybe something about "simplfying" the pipeline instead